### ISSUE TITLE
Is imgaug seeded by default?

### ISSUE THREAD

**Issue by CMCDragonkai** [aleju/imgaug#121]
I'm using imgaug in an augmentation script that takes a directory of images and creates a directory of augmented images.

I've noticed that every time I run this script, I get the same augmentations.

In order to get different augmentations, I have to change the seed: `ia.seed(some_number)`.

Thus in order to get random augmented images, I would have to seed it with a random number by default?

**Comment by aleju** [comment]
Yes, the library is configured by default to always use the same seed in order to make experiments more reproducible.
You can change that by calling `ia.seed()` as you said and providing it a random seed from e.g. numpy.

**Comment by jhaggle** [comment]
When I set ia.seed() I get
AssertionError: Expected argument 'entropy' or 'seedval' to be not-None, but bothwere None
Why is that?

**Comment by adriantre** [comment]
@aleju what do you think about having imgaug listen to the numpy.random.seed? i.e. if np.random.seed is reset, then imgaug will reset. In my case I am using the Pytorch-Lightning funcion [seed_everything](https://pytorch-lightning.readthedocs.io/en/latest/_modules/pytorch_lightning/utilities/seed.html#seed_everything) to set seeds, which ideally could seed everything :)
